Added few more commands

siva10x opened the pull request.

Commit:
Added few more commands

sridharrajs commented on SECURITY.md:
```diff
@@ -0,0 +1,2 @@
+## Security Best practices
+1. [https://snyk.io/blog/ten-npm-security-best-practices/](https://snyk.io/blog/ten-npm-security-best-practices/)
```
Can you please expand on the list here for the quick reference?

siva10x replied:
sure

Commit:
Enlisted few security best practices

siva10x commented:
Add few security best practices to SECURITY.md file [commitid](https://github.com/sridharrajs/node-cheatsheet/pull/1/commits/ed50d484307816032ced086335ce1bf340efa4ac)

sridharrajs commented on SECURITY.md:
```diff
@@ -1,2 +1,20 @@
+
+6. Embrace Linter security rules to identify and prevent vulnerabilities in the development phase itself. Use [tslint-config-security](https://www.npmjs.com/package/tslint-config-security), [eslint-plugin-security](https://github.com/nodesecurity/eslint-plugin-security)
+
+7. // Todo
```
todo?

siva10x replied:
I kept it as placeholder for adding future learnings. Removed it now as it is implict

Commit:
Update SECURITY.md

Kept it as a placeholder for future learning additions. But removing as it is implicit

sridharrajs reviewed (approved):
thanks for the PR @sivasankar-r
